src/suricata-common.h

```diff
@@ -314,6 +314,7 @@ typedef enum PacketProfileDetectId_ {
     PROF_DETECT_CLEANUP,
     PROF_DETECT_GETSGH,
     PROF_DETECT_MPM_FD_SMTP,
+    PROF_DETECT_MPM_TEMPLATE_BUFFER,
```

Comment:
This should have an accompanying util-profile.c update in PacketProfileDetectIdToString(). Otherwise it looks good. Was very helpful!